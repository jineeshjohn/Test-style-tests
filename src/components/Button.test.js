import React from 'react'
import styled from 'styled-components'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import Button from './Button';

test('it works again', () => {
  const tree = renderer.create(<Button />).toJSON()
  expect(tree).toMatchSnapshot()
  expect(tree).toHaveStyleRule('color', 'blue')
})