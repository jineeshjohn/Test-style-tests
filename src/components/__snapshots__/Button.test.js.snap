// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works again 1`] = `
.c0 {
  color: blue;
}

<button
  className="c0"
/>
`;
